src/index.ts

```diff
@@ -0,0 +1,15 @@
+import { createPlugin } from '@bluebase/core';
+import WebViewComp from './components/ReactNativeWebView';
+
+export default () => {
+	return createPlugin({
```

Comment:
have to tried to run and test? can you create Facebook app with this?

Reply:
Yes, I tested it on expo. It opens Facebook correctly. Verified by @danial24